handlers/list.ts

```diff
@@ -70,9 +70,9 @@ class ShoppingListHandler implements IShoppingList {
         this.gun.get('groups').get('groupId').get(this.groupId).get('members').open((data: any) => {
```

Comment:
if you put an actual reference to the user objects in members, the nester gun call, can be removed and the function can be simplified

Reply:
Read above tror jeg